### PR TITLE
MVP-240: Added Troubleshooting page to techdocs

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

  
[MVP-240](https://citz-imb.atlassian.net/jira/browse/MVP-240)

View the changes at https://dev.developer.gov.bc.ca/docs/default/component/citz-imb-sso-react-npm-package/using-the-package/troubleshooting/

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: techdocs/docs/using-the-package/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Diagnosing the Problem
+
+If an issue persists after following all the proper setup guides, follow these steps to help find the problem.
+
+1. In the `citz-imb-sso-express` package, you can get more detailed console logs by setting `DEBUG` and `VERBOSE_DEBUG` environment variables to `true`. These logs may help you spot an issue such as a null or undefined value.
+
+2. For your frontend you can check the Application cookies and Network tab in your browser's dev tools to help diagnose the problem.
+
+  - If you see an undefined `refresh_token` cookie, there is a problem.
```
Comment (SDToews) [inline techdocs/docs/using-the-package/troubleshooting.md:26]: "there is a problem" - what is it?  How do I fix it?

File: techdocs/docs/using-the-package/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - In the Network tab you will see a chain of login requests and a `?refresh_expires_in=1799&post_login_redirect_url=` request. If `refresh_expires_in` is `undefined` there is a problem.
```
Comment (SDToews) [inline techdocs/docs/using-the-package/troubleshooting.md:27]: "there is a problem" - what is it?  How do I fix it?

**Review by SDToews — CHANGES_REQUESTED** [review]
Please indicate how to solve the problem on your troubleshooting page

File: techdocs/docs/using-the-package/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Troubleshooting
+
+Sometimes issues may arise when using the packages for the first time, or when updating to a new version. Use this guide to help troubleshoot the problem.
+
+## Initial Setup Checklist
+
+1. Ensure you follow the Quick Start Guides closely for each package.
+
+  - [SSO React - Quick Start Guide](../getting-started/quick-start-guide.md)
+  - [SSO Express - Quick Start Guide](https://developer.gov.bc.ca/docs/default/component/citz-imb-sso-express-npm-package/getting-started/quick-start-guide/)
```
Comment (TaylorFries) [inline techdocs/docs/using-the-package/troubleshooting.md:10]: this link doesn't seem to work

File: techdocs/docs/using-the-package/troubleshooting.md (hunk 1 above)
Comment (BradyMitch) [reply techdocs/docs/using-the-package/troubleshooting.md:26]: Its not entirely clear what the problem is but I can add a bit more of a description.

File: techdocs/docs/using-the-package/troubleshooting.md (hunk 3 above)
Comment (BradyMitch) [reply techdocs/docs/using-the-package/troubleshooting.md:10]: It wont work until this is in production.

**Review by SDToews — APPROVED** [review]

**Pull request merged**